application/src/main/java/com/sanction/thunder/resources/UserResource.java

```diff
@@ -114,51 +123,43 @@ public Response updateUser(@Auth Key key, PilotUser user) {
   /**
    * Retrieves a PilotUser from the database.
    *
-   * @param username The username of the user to retrieve.
-   * @return The user that was found in the database.
+   * @return The pilotUser that was found in the database.
    */
   @GET
-  public Response getUser(@Auth Key key, @QueryParam("username") String username) {
+  public Response getUser(@Auth User user) {
```

Comment:
I know why you took out the `QueryParam`, but something feels wrong now. It might be because now a GET request looks like:

```
GET thunder.sanctionco.com/users
```

and returns the user that authenticated. If you just read that off, it seems weird that `/users` would just return one user. Compare that to before:

```
GET thunder.sanctionco.com/users?username=Testy
```

The `QueryParam` qualifies the URL and it makes more sense just by looking at it.

We should talk about this and see what we want to do.